src/event/helpers.ts

```diff
@@ -0,0 +1,31 @@
+import { CallbackContainer, Listener } from './interfaces';
+
+/** @hidden */
```

Comment:
What does `@hidden` mean?

Reply:
It means don't render typedocs for it.  Not sure what the difference is between `@hidden` and `@ignore`.
https://typedoc.org/guides/doccomments/#hidden-and-ignore